x/concentrated-liquidity/fees.go

```diff
@@ -214,7 +214,7 @@ func (k Keeper) collectFees(ctx sdk.Context, poolId uint64, owner sdk.AccAddress
 	}
```

Comment:
I believe the lines above this (210-213) need to be removed. We are properly accounting for the current fee growth outside when we use the ClaimRewardsCustomAcc func, so updating the accumulator prior that causes issues.